server/@types/ui/index.d.ts

```diff
@@ -87,6 +87,13 @@ type ReferralSharedPageData = {
   submissionSummaryListRows: Array<GovukFrontendSummaryListRowWithKeyAndValue>
 }
 
+type ReferralSummaryDetails = {
+  personName: Person['name']
```

Comment:
Do we want to remove the `personName` here if we're getting this from the backend?